src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -231,8 +257,10 @@ protected final void verifyWithInlineConfigParser(String filePath, String... exp
             throws Exception {
         final TestInputConfiguration testInputConfiguration =
                 InlineConfigParser.parse(filePath);
+        final DefaultConfiguration treeWalker = processTreeWalkerModuleFromFile(
+                testInputConfiguration);
         final DefaultConfiguration parsedConfig =
-                testInputConfiguration.createConfiguration();
+                testInputConfiguration.createConfiguration(treeWalker);
```

Comment:
Can you avoid such change in this test method ?
Just load Treewalker as another module, if it already exists, put properties from  file to existing module

Reply:
You can put Treewalker before check in file to make more natural

Reply:
Done